Documentation/gettingstarted/hubble-install.rst

```diff
@@ -7,16 +7,10 @@
       .. parsed-literal::
 
          export HUBBLE_VERSION=$(curl -s https://raw.githubusercontent.com/cilium/hubble/master/stable.txt)
-         curl -LO "https://github.com/cilium/hubble/releases/download/$HUBBLE_VERSION/hubble-linux-amd64.tar.gz"
-         curl -LO "https://github.com/cilium/hubble/releases/download/$HUBBLE_VERSION/hubble-linux-amd64.tar.gz.sha256sum"
+         curl -L --remote-name-all https://github.com/cilium/hubble/releases/download/$HUBBLE_VERSION/hubble-linux-amd64.tar.gz{,.sha256sum}
          sha256sum --check hubble-linux-amd64.tar.gz.sha256sum
-         tar zxf hubble-linux-amd64.tar.gz
-
-      and move the ``hubble`` CLI to a directory listed in the ``$PATH`` environment variable. For example:
-
-      .. parsed-literal::
-
-         sudo mv hubble /usr/local/bin
+         sudo tar xzvfC hubble-linux-amd64.tar.gz /usr/local/bin
+         rm hubble-linux-amd64.tar.gz hubble-linux-amd64.tar.gz.sha256sum
```

Comment:
Nit (but not blocking, also find as it is)
```suggestion
         rm hubble-linux-amd64.tar.gz{,.sha256sum}
```

Reply:
Nice catch, fixed in latest push.